mysite/models/models.py

```diff
@@ -1995,6 +2038,10 @@ def _calculate_w4_cooking_max_plate_levels(self):
         self.cooking['CurrentRemainingMeals'] = (self.cooking['MealsUnlocked'] * self.cooking['PlayerMaxPlateLvl']) - self.cooking['PlayerTotalMealLevels']
         self.cooking['MaxRemainingMeals'] = (maxMeals * maxMealLevel) - self.cooking['PlayerTotalMealLevels']
 
+    def _calculate_w4_meal_multi(self):
+        for meal in self.meals:
```

Comment:
* Please preface with a check for the Jewel being enabled.
`if self.labJewels["Black Diamond Rhinestone"]["Enabled"]:`